addons/l10n_it/data/account_tax_template.xml

```diff
@@ -140,7 +140,7 @@
     </record>
     <!-- Purchase of Goods (Acquisti Merce) ................................................... -->
     <record id="22am" model="account.tax.template">
-        <field name="description">22am</field>
+        <field name="description">22PG</field>
```

Comment:
```suggestion
        <field name="description">22%</field>
```
The description should only mention the rate (seen with TSB).
--> Several occurrences.